src/js/OGVPlayer.js

```diff
@@ -895,6 +901,10 @@ class OGVPlayer extends OGVJSElement {
 		let audioFeeder = this._audioFeeder = new AudioFeeder(audioOptions);
 		audioFeeder.init(this._audioInfo.channels, this._audioInfo.rate);
 
+		//Fire when _audioFeeder is populated
+		if (this._audioFeeder)
```

Comment:
I think that should be checking the event callback rather than the `_audioFeeder` property:

```suggestion
		if (this.onaudiofeedercreated)
```

With that fix in I think it'll be good to go.